.github/workflows/ci.yml

```diff
@@ -14,9 +14,6 @@ on:
       - '**.md'
       - 'LICENSE'
   pull_request:
-    paths-ignore:
```

Comment:
Those are only for pushes to the main branch, so that should only take effect after the merge. If only those text files change, the workflow will still run for PRs (including pushes to the PRs), but won't run on the main branch after it has been merged.

Reply:
Ah ok. sgtm